final/MealsharingNet/Program.cs

```diff
@@ -0,0 +1,34 @@
+using MealsharingNet;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
+
+builder.Services.AddControllers();
+builder.Services.AddSingleton<IMealRepository, MealRepository>();
+builder.Services.AddSingleton<IReservationRepository, ReservationRepository>();
+builder.Services.AddSingleton<IReviewRepository, ReviewRepository>();
+// builder.Services.AddSingleton<IUserRepository, UserRepository>();
```

Comment:
Any reason why you chose to comment this part? 😊

Reply:
I forget it here :blush: